docs/commands/preview.md

```diff
@@ -4,9 +4,7 @@
 
 The `preview` command starts a local preview server for a Redocly project. Use the preview server to develop your project locally before deployment.
 
-{% admonition type="warning" name="Pre-release" %}
-This command is for our pre-release products, currently open for early access to a small number of users. Announcements about the release are made through our [mailing list](https://redocly.com/product-updates/).
-{% /admonition %}
+{% partial file="../_snippets/new-product-alert-commands.md" %}
```

Comment:
As we're working up to release, I think we can drop this admonition completely.

Reply:
I'd like to discuss keeping something attention-grabbing since `eject` and `translate` are Realm-specific. 

Some type of visual distinction in the same place on each respective command doc would be useful and could train users where to look to see which tech a command works with. If admonitions are too heavy, something like badges, separators, etc.?

Reply:
Push is sort of realm specific except that push isn't. We'll have the same thing with spot when it lands I think. Adding a note is good though (I'm just thinking about the same thing while reviewing the translation PR)

Reply:
Added a statement at bottom of intro. Will do the same in PR for `translate`.